apps/backend/api/src/controllers/patient.accommodation.controller.ts

```diff
@@ -55,28 +55,28 @@ export const addRelatedAccommodationHandler = async (
   try {
     const patient = await prisma.patient.findUnique({
       where: {
-        id: req.params.id,
+        id: req.body.id,
```

Comment:
No.

Reply:
It's a post request. Why wouldn't we use the body?

Reply:
because it's not the convention

Reply:
https://www.mscharhag.com/api-design/rest-many-to-many-relations

Reply:
the first solution was used everywhere

Reply:
Alright. Feel free to fix it, or I will do it tomorrow